src/language-services/inspectionUtils.ts

```diff
@@ -92,6 +95,24 @@ export function getSymbolsForLetExpression(expressionNode: PQP.Language.Ast.LetE
     return documentSymbols;
 }
 
+export function getSymbolsForRecord(
+    recordNode: PQP.Language.Ast.RecordExpression | PQP.Language.Ast.RecordLiteral,
+): DocumentSymbol[] {
+    const documentSymbols: DocumentSymbol[] = [];
```

Comment:
nit: this could be replaced with a `recordNode.content.elements.reduce(...)`. It's arguable if that makes it more/less readable.